### PR TITLE
fix: add case-insensitive filtering in all searchs

### PR DESCRIPTION
**Pull request by brunooomelo**
reference issue #99

### CODE REVIEW

File: src/modules/broadcasting/repositories/in-memory/InMemoryMessagesRepository.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class InMemoryMessagesRepository implements IMessagesRepository {
 
     if (query) {
       messageList = this.items.filter(message =>
-        message.subject.value.includes(query)
+        message.subject.value.includes(query.toLowerCase())
```
Comment (diego3g) [inline src/modules/broadcasting/repositories/in-memory/InMemoryMessagesRepository.ts:55]: Shoudn't we also lowercase the `message.subject.value` as well before comparing to query? As we are only lowercasing the query, if the subject contain capitalized chars, it won't be found.

File: src/modules/broadcasting/useCases/SearchMessages/SearchMessagesController.spec.ts [hunk 2]
```diff
@@ -66,4 +67,29 @@ describe('Search Messages (e2e)', () => {
       ])
     )
   })
+  it('should be able to search messages', async () => {
```
Comment (diego3g) [inline src/modules/broadcasting/useCases/SearchMessages/SearchMessagesController.spec.ts:70]: I think the test name here is duplicated.

File: src/modules/subscriptions/useCases/SearchContacts/SearchContacts.spec.ts [hunk 3]
```diff
@@ -40,6 +40,13 @@ describe('Search Contacts', () => {
     expect(response.totalCount).toEqual(1)
     expect(response.data[0].name.value).toEqual('John Doe5')
   })
+  it('should be able to search through contacts', async () => {
```
Comment (diego3g) [inline src/modules/subscriptions/useCases/SearchContacts/SearchContacts.spec.ts:43]: Again, the test name is duplicated.

**Review by diego3g — CHANGES_REQUESTED** [review]

**Comment by diego3g** [comment]
@brunooomelo will u be able to fix this?

**Comment by brunooomelo** [comment]
Sure.

**Review by diego3g — APPROVED** [review]

**Pull request merged**